seaborn/utils.py

```diff
@@ -414,3 +414,9 @@ def axes_ticklabels_overlap(ax):
     """
     return (axis_ticklabels_overlap(ax.get_xticklabels()),
             axis_ticklabels_overlap(ax.get_yticklabels()))
+
+
+def check_mpl_version():
+    import matplotlib as mpl
+    major, minor, point = tuple(map(float, mpl.__version__.split('.')))
```

Comment:
It's good because it knows about things like `.dev`, `.rc1` as minor version, etc, which otherwise would, I think, break here.